test/data/pdss_hkft.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0"?>
+<ctml>
+  <validate reaction="yes" species="yes"/>
+  <!-- Clearly, using an IdealGas-type phase is not appropriate
+       for a liquid water solvent. However, it seems to load
+       correctly and I needed a place to test the conversion of
+       these phase thermo types so \shrug
+  -->
+  <phase dim="3" id="vpss_gas_pdss_hkft_phase">
+    <elementArray datasrc="elements.xml">
+      Na Cl H O
+    </elementArray>
+    <speciesArray datasrc="#species_data">
+      H2O(L) Na+ Cl- H+ OH-
+    </speciesArray>
+    <thermo model="IdealGasVPSS"/>
```

Comment:
The `IdealMolalSolution` model might be a more reasonable choice.

Reply:
This choice was intentional to exercise this particular thermo model, not for physical correctness. That it works at all (i.e., that the `IdealGasVPSS` class doesn't throw because of the solvent being water) is perhaps a separate issue that should be addressed...